app/views/content_items/_detailed_guide_b.html.erb

```diff
@@ -0,0 +1,54 @@
+<%= content_for :title, @content_item.page_title %>
```

Comment:
It's not clear what the differences between `a.html.erb` and `b.html.erb` are. Rather than two templates, could the variations be provided by picking different partials? This reduces duplication and makes the template variations clearer.